### ISSUE TITLE
New XAPI plugin - get an host lsblk output

### ISSUE THREAD

**Issue by benjamreis** [xapi-project/xen-api#4202]
Hello, I've written a new XAPI plugin in order to get the output of lsblk command on a host.
an be used by a XAPI client to get a json object describing the ouput of a `lsblk` command on a host.

We want to use it in XO to view the disks and partitions on a host.

For instance when creating a SR on a local storage it can be used to select where to create the SR in the local storage

Can I commit here? I suppose it'd go in `script/plugins` since it can be called by `xe call-plugin`

Thanks!

**Comment by robhoes** [comment]
If XO need such plugins, then perhaps they can be packaged and installed separately from xapi?

**Comment by benjamreis** [comment]
Yeah we could, but I figured perhaps other XAPI clients might want to use it nand/or there might be a standard repo where to store XAPI plugins.

**Comment by olivierlambert** [comment]
@robhoes should you create a 3rd party XAPI plugin repo on your organization, or should we do that on XCP-ng Github project org?

**Comment by robhoes** [comment]
It is probably easier for you to maintain them on your org?

**Comment by stormi** [comment]
It's easy for us to maintain them on our org, but they're not specific to XCP-ng, that's why it would seem more logical to have them under the xapi-project org, with any required warning addressed to uses, such as "those plugins are contributed by the community and provided as is. Use at your own risk". We could of course take a significant part in the maintenance of this repository.

**Comment by benjamreis** [comment]
for now we'll store the plugin on our repo: https://github.com/xcp-ng/xcp-ng-xapi-plugins/pull/18

We'd gladly move it at yours if someday the feature is of interest for your users. :)
